erigon-lib/state/kv_temporal_copy_test.go

```diff
@@ -0,0 +1,271 @@
+// Copyright 2024 The Erigon Authors
```

Comment:
should this file be removed?

Reply:
Can’t use temporal package from state package - circular dependency. To break: need to move SD to another package. SD has biz-logic, temporal.tx doesn’t.